packages/playground/blueprints/src/lib/steps/activate-plugin.ts

```diff
@@ -75,6 +78,12 @@ export const activatePlugin: StepHandler<ActivatePluginStep> = async (
 		},
 	});
 	if (activatePluginResult.text) {
+		/**
+		 * If the plugin was successfully activated, we can return early.
+		 */
+		if (activatePluginResult.text === 'true') {
+			return;
+		}
```

Comment:
This isn't directly related to this PR, but while I was debugging it, I realized that this will make the `activatePlugin` step more robust and will avoid the extra check when it's not needed.

Reply:
On it's own this fix would allow the PR test to pass because the plugin would successfully activate inside the `plugins` folder.